edk2-rockchip/Platform/Rockchip/Rk356x/AcpiTables/Mshc.asl

```diff
@@ -33,4 +33,42 @@ Device (MSH0) {
         })
         Return (RBUF)
     }
-}
\ No newline at end of file
+}
+
+// Mobile Storage Host Controller
```

Comment:
This will cause problems on any board that hasn't set up the device. We need to make this conditional.

(I'd really like to see this done as a separate change)